content_script.js

```diff
@@ -0,0 +1,10 @@
+function handleMessage(request) {
+    pauseVideo() //we don't care about message contents, pause on any message received
+};
+
+function pauseVideo() {
```

Comment:
Looks good! `ff2mpv` can technically work with `<audio>` too, but let's just not worry about that for the time being.